packages/terra-demographics-banner/src/terra-dev-site/doc/demographics-banner/UpgradeGuide.3.doc.mdx

```diff
@@ -4,6 +4,14 @@ import { Badge } from 'terra-demographics-banner/package.json?dev-site-package';
 
 # Terra Demographics Banner Upgrade Guide
 
+## Changes from version 3 to version 4
+
+### Breaking Changes
+* Removed `gender`. Replaced by `administrativeSex`
+
+### New props
+* Added `administrativeSex`
```

Comment:
Shouldn't the propType changes be mentioned here too?

Reply:
Updated: https://github.com/cerner/terra-core/pull/2856/commits/5b98d5ff7de9b4397bea7b94cb00934e5930d1e5